source/develop/release-management/features/network/allowExplicitDnsConfiguration.html.md

```diff
@@ -0,0 +1,177 @@
+engine needs to comply to this limitation as well.
+
+### REST
+
+As mentioned, you can specify DNS Configuration at two places.
+Corresponding REST areas will be altered. But first we need to add new
+element 'dns_configuration' as:
+ 
+```
+@Type
+public interface DnsResolverConfiguration {
+ String[] nameServers();
+}
+```
+
+…, which will be translated into following XSchema definition during
+engine build.
+
+
+```
+<xs:element name="dns_resolver_configuration" type="DnsResolverConfiguration"/>
+
+  <xs:complexType name="DnsResolverConfiguration">
+    <xs:sequence>
+      <xs:element maxOccurs="1" minOccurs="0" name="name_servers">
+        <xs:complexType>
+          <xs:annotation>
+            <xs:appinfo>
+              <jaxb:class name="NameServersList"/>
+            </xs:appinfo>
+          </xs:annotation>
+          <xs:sequence>
+            <xs:element maxOccurs="unbounded" minOccurs="0" name="name_server" type="xs:string">
+              <xs:annotation>
+                <xs:appinfo>
+                  <jaxb:property name="NameServers"/>
+                </xs:appinfo>
+              </xs:annotation>
+            </xs:element>
+          </xs:sequence>
+        </xs:complexType>
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+```
+
+After doing so, we're prepared to use following xml fragment defining
+DNS configuration in several places (read on):
+
+```
+<dns_resolver_configuration>
+  <name_servers>
+    <name_server>1.1.1.1</name_server>
+    <name_server>2.2.2.2</name_server>
+  </name_servers>
+</dns_resolver_configuration>
+```
+
+#### Updating (Management) Network
+You can update any network with DNS Configuration, however all such DNS
+Configurations will be simply ignored during creation of new host except 
+for the DNS configuration defined on the network, which happens to be 
+management network at time of creating new host.
+
+```
+PUT /ovirt-engine/api/networks/{network:id} HTTP/1.1
+Content-Type: application/xml
+
+<network>
+  <dns_resolver_configuration>
+    <name_servers>
+      <name_server>1.1.1.1</name_server>
+      <name_server>2.2.2.2</name_server>
+    </name_servers>
+  </dns_resolver_configuration>
+</network>
+```
+
+#### Updating 'network attachment' of Management Network on specific Host 
+Only the network attachment of the management network can be updated
+with DNS configuration.
+
+```
+PUT ovirt-engine/api/hosts/{host:id}/networkattachments/{networkattachment:id} HTTP/1.1
+Content-type: application/xml
+
+<network_attachment>
+  <dns_resolver_configuration>
+    <name_servers>
+      <name_server>1.1.1.1</name_server>
+      <name_server>2.2.2.2</name_server>
+    </name_servers>
+  </dns_resolver_configuration>
+</network_attachment>
+
+```
+
+### GUI
+
+As mentioned, you can specify DNS Configuration at two places:
+
+#### Updating (Management) Network
+You can update any network with DNS Configuration, however all such DNS
```

Comment:
discuss it with edward, who thinks opposite and is against what you've suggested.